app/Http/Controllers/RankingController.php

```diff
@@ -364,51 +378,97 @@ public function kudosu()
         return ext_view('rankings.kudosu', compact('scores'));
     }
 
-    public function spotlight($mode)
+    private function maxResults(int $rulesetId, ?CountryStatistics $countryStats, Builder $stats, array $params): int
     {
-        $chartId = get_int($this->params['spotlight'] ?? null);
+        switch ($params['type']) {
```

Comment:
missing `default` return/throw

Reply:
🤔 php will throw magically due to missing return?

Reply:
good point 🤔
guess we can deal with static analysis problem later (so never)